presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -126,6 +126,10 @@ Operations
 
     Returns the geometry value that represents the point set symmetric difference of two geometries.
 
+.. function:: ST_Union(Geometry, Geometry) -> Geometry
+
+    Returns a geometry that represents the point set union of the Geometries.
```

Comment:
s/of the Geometries/of the input geometries

Reply:
Could you mention that GEOMETRYCOLLECTION is not supported?

`This function doesn't support geometry collections.`